src/marks/waffle.js

```diff
@@ -200,12 +276,12 @@ function spread(domain) {
   return max - min;
 }
 
-export function waffleX(data, options = {}) {
+export function waffleX(data, {tip, ...options} = {}) {
   if (!hasXY(options)) options = {...options, y: indexOf, x2: identity};
-  return new WaffleX(data, maybeStackX(maybeIntervalX(maybeIdentityX(options))));
+  return new WaffleX(data, {tip, ...maybeStackX(maybeIntervalX(maybeIdentityX(options)))});
```

Comment:
What is this change doing? It’s not covered by tests, and I don’t notice any difference in manual testing.

Reply:
Ah, so it’s removing the default behavior of the stack transform when the **tip** option is true, which would be to use `pointer: "y"` on on waffleX, and `pointer: "x"` on waffleY. I get why you’re doing it, but I’m not sure it’s an improvement — in a way it makes things worse by having lots of dead space where the tip isn’t activated even though you’re hovering over a waffle; you need to be within 40px of the waffle centroid for it to activate, but it isn’t always obvious where that centroid is, and why the tip isn’t activating.

I wonder if this means we should set the **maxRadius** to Infinity for waffle marks (just so it’s clear that something is happening), or if it means the overall approach of choosing a single representative point for a mark is insufficient, and we need to supply the pointer transform with a full geometry so it can do _e.g._ a signed distance check.

Reply:
Perhaps another possibility is to change the pointer behavior when _x_ (or _y_) is a band scale. But that’s a bit weird because the provided _x_ here is already in screen space…

Reply:
I think setting maxRadius to infinity is the correct interim choice. But it would be neat to have a full geometry (both for geo and waffles) in the future.

Note that, for the geo mark, one thing that works well currently with a centroid-based pointer — and will be harder to solve with full geometry —, is to make "small countries" discoverable even when their shape is very small (islands are easy, the hard part is small countries surrounded by larger countries).

Reply:
Hmm, I’m not sure how we’d set **maxRadius** to Infinity for waffle marks. We can do it on the **tip** option for the derived tip, but that wouldn’t affect how the pointer transform behaves when applied to the waffle mark. 🤔 It kind of makes me think the waffle mark should supply a _geometry_ channel, and then the pointer transform should compute the signed distance to the geometries… (That would also let us fix pointing at bars wider than 40px, too.)